### PR TITLE
change css file name & remove front.css : should not be under cvs.

### PR DESCRIPTION
**Pull request by SebSept**

### CODE REVIEW

File: upgrade/upgrade-1.1.0.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+/**
+ * /views/css/front.css is under cvs, it should not be.
+ * It's shop specific.
+ * It must not be overwritten by module upgrade.
+ *
+ * Copy current file contents to new file
+ */
+function upgrade_module_1_1_0($module)
+{
+    // @todo use dirname(__FILE__) instead of __DIR__ ?
```
Comment (nenes25) [inline upgrade/upgrade-1.1.0.php]: The process is ok for me can we remove the @todo ?

**Review by nenes25 — CHANGES_REQUESTED** [review]
Just remove the @todo 
wdyt about adding a .gitignore file ?

File: upgrade/upgrade-1.1.0.php (hunk 1 above)
Comment (SebSept) [reply upgrade/upgrade-1.1.0.php]: yes, I'll add a .gitignore file with views/css/fop_customcss.css

**Review by nenes25 — APPROVED** [review]
Seems good for me , thanks @SebSept  !

**Comment by SebSept** [comment]
I'll run a test later, then we could merge to dev.

**Review by PrestaSafe — COMMENTED** [review]
GG

**Pull request merged**